src/analytics/lambdas/scan-metadata-workflow/update-workflow-info.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import {
+  DynamoDBClient,
+} from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient, PutCommand, PutCommandInput } from '@aws-sdk/lib-dynamodb';
+import { logger } from '../../../common/powertools';
+import { aws_sdk_client_common_config } from '../../../common/sdk-client-config';
+
+const workflow_info_ddb_table = parseDynamoDBTableARN(process.env.WORKFLOW_INFO_DDB_TABLE_ARN!);
+
+const metadata_info_ddb_table = parseDynamoDBTableARN(process.env.METADATA_DDB_TABLE_ARN!);
+
+const workFlowDDBClient = new DynamoDBClient({
+  ...aws_sdk_client_common_config,
+});
+
+const workFlowDDBDocClient = DynamoDBDocumentClient.from(workFlowDDBClient);
+
+const metaDataDDBClient = new DynamoDBClient({
+  ...aws_sdk_client_common_config,
+  region: metadata_info_ddb_table.ddb_region,
```

Comment:
align the variable names `ddb_region` to others like `lastScanEndDate`

Reply:
Refactored it.